test/organizations/members_test.exs

```diff
@@ -20,7 +20,8 @@ defmodule Tentacat.Organizations.MembersTest do
 
   test "member?/3" do
     use_cassette "members#member_" do
-      assert member?("elixir-conspiracy", "josephwilk", @client) == {302, nil}
+      {status_code, _} = member?("elixir-conspiracy", "josephwilk", @client)
+      assert status_code == 404
```

Comment:
@duksis since now we are redirecting automatically, the result of this action had to change, this will "brake" compatibility but in my opinion makes more sense, WDYT ?

Reply:
by "brake" you mean the fact that it won't return the `30x` statuses anymore?

Reply:
Yes, I mean the developers that are  handling themselves the 302 (not sure is happening)

Reply:
even for those It should not be a breaking change.